cypress/support/step_definitions/context.js

```diff
@@ -4,19 +4,15 @@ Before(() => {
   cy.context = {};
 });
 
-Given('I clear cache', () => {
+Given('I clear cache', async () => {
```

Comment:
why use `async` without an `await` ?

Reply:
Forgot to remove, thank's for the report, done :)